src/arch/xtensa/CMakeLists.txt

```diff
@@ -508,13 +508,20 @@ add_custom_target(
 	USES_TERMINAL
```

Comment:
Add some prefix, "cmake:" (used) or "build:" (doesn't seem to be used).

Reply:
Sorry, not following your comment ? I guess it can be updated incrementally though.

Reply:
@kv2019i I don't get your comment either sorry.

Reply:
@lgirdwood @marc-hb I referred to the first line of the git commit message of the first patch:

"Really copy sof-$platform.{ri,ldc} to sof.{ri,ldc} "

We would typically have some prefix like "build:" or "cmake:" for all patches. This one seemed really out-of-place.

But style only, merge is ok.

Reply:
Thanks @kv2019i , I got confused because this comment is pointing at line 508 in the code and I added a similar line so I didn't think about the commit message(s) sorry.

Github does not care about commit messages, that's why we can't review them. Their preferred review model is no force-push and a possibly long stream of fixup commits 
https://github.com/zephyrproject-rtos/zephyr/pull/14444#issuecomment-575320941